src/alire/alire-dependencies-states.ads

```diff
@@ -102,6 +96,11 @@ package Alire.Dependencies.States is
 
    function Is_Pinned (This : State) return Boolean;
 
+   function Is_Provided (This : State) return Boolean;
+   --  True when the release name is different from the dependency crate
+
+   function Is_Shared (This : State) return Boolean;
```

Comment:
Should it be `Is_Installed`?

Reply:
I started using the idea of 'shared releases' internally, as these are just regular releases in a shared location. But it can be changed if confusing (likewise then for the `Alire.Shared` package).

Reply:
It will depend on how and if we implement an `alr install` I guess. We can keep shared for now, up to you.

Reply:
My current mood is to avoid `install` entirely. I was thinking of using `alr share` for non-toolchain crates shared in the configuration, and `alr export` for the `gprinstall-in-common-prefix` idea. So we convey that exported crates are something the user wants to use out of Alire. But I'm easy to sway on these naming things.

For now if you don't mind I'll leave it as is then.

Reply:
If the user wants to make some application available system-wide, I would say the usual keyword used by package managers and build systems is `install` instead of `export`. `export` sounds like an action to extract data from an application.

Reply:
In this case we would be "extracting" binaries from the Alire ecosystem. But I see your point. `alr share` for shared dependencies across workspaces and `alr install` for deploying crates to a common prefix can also work for me.

Another possibility is to have it under `get`, e.g., `alr get --install`. The operation in itself is essentially an `alr get + alr build + gprinstall`.